source/includes/documentation/_vue_js_elements.md

```diff
@@ -2631,41 +2631,59 @@ __Progress circular__ - The v-progress-circular component is used to convey data
 
 #### 5.27.2 Progress linear
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/progress-linear/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/progress-linear/)
 
+Progress linear is located in the following class:
 - __Java__: _package com.epam.jdi.light.vuetify.elements.common.ProgressBar.java_
 
 ```java
-    @Test(dataProvider = "bufferValueProgressBarsTestsDataProvider",
-      dataProviderClass = ProgressBarsDataProvider.class)
-    public void bufferValueProgressBarsTests(int index, String color) {
-      bufferValueProgressBars.get(index).is().displayed();
-      bufferValueProgressBars.get(index).is().reactive();
-      bufferValueProgressBars.get(index).is().determinate();
-      bufferValueProgressBars.get(index).has().color(color);
-      bufferValueProgressBars.get(index).click();
-      bufferValueProgressBars.get(index).has().value(50.0);
-    }
-
+@Test(description = "Test checks absolute value of progress linear and checks if it is hidden : value")
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby
![image](https://user-images.githubusercontent.com/55189259/220054419-1b0bf8a1-4d2a-4971-a29a-0fc8e17ffec2.png)

Reply:
added